webpack.config.js

```diff
@@ -6,7 +6,7 @@ const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const webpack = require("webpack");
 
 let og_image_url = process.env.RIOT_OG_IMAGE_URL;
-if (!og_image_url) og_image_url = 'https://riot.im/app/themes/riot/img/logos/riot-im-logo-black-text.png';
+if (!og_image_url) og_image_url = 'https://about.riot.im/images/opengraph.png';
```

Comment:
Perhaps it's better to keep the app self-contained by copying this asset into the logos folder? If we redesign about.riot.im, some day, we're likely to forget we've depended on this path.